src/CompanyService.Mappers/Models/DepartmentUserInfoMapper.cs

```diff
@@ -11,15 +11,21 @@ public class DepartmentUserInfoMapper : IDepartmentUserInfoMapper
     {
         private readonly IPositionInfoMapper _positionInfoMapper;
         private readonly IAccessValidator _accessValidator;
+        private readonly IImageInfoMapper _imageMapper;
 
-        public DepartmentUserInfoMapper(IPositionInfoMapper positionInfoMapper, IAccessValidator accessValidator)
+        public DepartmentUserInfoMapper(
+            IPositionInfoMapper positionInfoMapper,
+            IAccessValidator accessValidator,
+            IImageInfoMapper imageMapper)
         {
             _positionInfoMapper = positionInfoMapper;
             _accessValidator = accessValidator;
+            _imageMapper = imageMapper;
         }
 
         public UserInfo Map(UserData userData, DbPositionUser dbPositionUser, ImageData image)
         {
+            //I don't understand this
             if (userData == null || (dbPositionUser == null && !_accessValidator.IsAdmin(userData.Id)))
```

Comment:
Do you think we should remove dbPositionUser == null && !_accessValidator.IsAdmin(userData.Id)?

Reply:
I removed it